main/src/ca/uwaterloo/flix/language/phase/Resolver.scala

```diff
@@ -3235,8 +3255,8 @@ object Resolver {
   /**
     * Returns the class and field reflection objects for the given `className` and `fieldName`.
     */
-  private def lookupJvmField(className: String, fieldName: String, static: Boolean, loc: SourceLocation)(implicit flix: Flix): Result[(Class[_], Field), ResolutionError with Recoverable] = {
-    lookupJvmClass(className, loc).flatMap {
+  private def lookupJvmField(className: String, fieldName: String, static: Boolean, loc: SourceLocation, ns0: Name.NName, root: NamedAst.Root)(implicit flix: Flix): Result[(Class[_], Field), ResolutionError with Recoverable] = {
```

Comment:
root should be implicit. Flix and root are always implicit because they are so common.